sql/org_mozilla_firefox_derived/event_types_v1/query.sql

```diff
@@ -0,0 +1,243 @@
+WITH current_events AS (
```

Comment:
I understand this query is a _pain_ to look at. A bit of explanation here may be in order.

We're basically doing exactly what's in the `init` query (see the `per_event_property` and `per_event` CTEs). Ahead of that, we're stripping out known events, event_properties, and event_property_values; we then create the new values for each and `UNION` them to the existing ones (see e.g.`all_primary_event_types`). Once we have the three different sets of data (_all_ events, event_properties, and event_property_values) we do do exactly what we do in the init function.